docs/installation/upgrade/include/2.4-removed-settings.inc

```diff
@@ -0,0 +1,44 @@
+| `disable_plaintext_auth` | Replaced by [[setting,auth_allow_cleartext]]. |
+| `imap_id_log` | Replaced by the [[event,imap_id_received]] event. |
+| `login_access_sockets` | Use [[link,auth_lua]] instead. Dovecot will fail to start if this setting is present in configuration. |
+| `push_notification_backend` | Use [[setting,push_notification_driver]] instead. |
+| `quota_set` | |
+| `sieve_dir` | See [[link,sieve_file]] and [[link,sieve_location]]. |
+| `sieve_global_dir` | See [[link,sieve_file]] and [[link,sieve_location]]. |
+| `sieve_global_path` | See [[link,sieve_file]] and [[link,sieve_location]]. |
+| `sieve_vacation_max_subject_codepoints` | |
+| `auth_policy_server_timeout_msecs` | See: [[link,auth_policy_configuration]]. |
+| `mail_crypt_require_encrypted_user_key` | Replaced by [[setting,crypt_user_key_require_encrypted]], [[setting,crypt_user_key_password]] and [[setting,crypt_user_key_encryption_key]]. |
+| `verbose_ssl` | Use [[setting,log_debug,category=ssl]] instead. |
+| `mail_location` setting & `mail` userdb field | Split into multiple [[link,mail_location,mail_*]] settings. |
+| `listescape` plugin | [[setting,mailbox_list_storage_escape_char]] setting. |
+| `fts` | Replaced by [[setting,fts]] named filter. |
+| `fts_autoindex_exclude` | Replaced by boolean property [[setting,fts_autoindex]], nested inside [[setting,mailbox]] blocks. Note that the values are inverted as compared to those in the old `fts_autoindex_exclude`. |
+| `fts_decoder` | Replaced by [[setting,fts_decoder_driver]] and [[setting,fts_decoder_script_socket_path]]. |
+| `fts_solr` | Replaced by [[setting,fts_solr_url]], [[setting,fts_solr_batch_size]], [[setting,fts_solr_soft_commit]], and [[setting,http_client_rawlog_dir]] settings. |
+| `fts_tika` | Replaced by [[setting,fts_decoder_driver]] and [[setting,fts_decoder_tika_url]]. |
+| `fts_language_config` | Renamed to [[setting,textcat_config_path]]. |
+| `fts_languages` | Converted into [[setting,language]] blocks. |
+| `fts_filters` | Split into [[setting,language_filters]], [[setting,language_filter_normalizer_icu_id]], [[setting,language_filter_stopwords_dir]]. |
+| `fts_tokenizers` | Split into [[setting,language_tokenizers]], [[setting,language_tokenizer_address_token_maxlen]], [[setting,language_tokenizer_generic_algorithm]], [[setting,language_tokenizer_generic_token_maxlen]], [[setting,language_tokenizer_generic_wb5a]], [[setting,language_tokenizer_kuromoji_icu_id]], [[setting,language_tokenizer_kuromoji_split_compounds]], [[setting,language_tokenizer_kuromoji_token_maxlen]]. |
+| `service_count` | Renamed to [[setting,service_restart_request_count]]. The default value is set to `unlimited`. Value `0` is now a configuration error. |
+| passdb/userdb `:protected` | Renamed to `:default` |
+| `passdb_default_fields`, `passdb_override_fields` | Replaced by [[setting,passdb_fields]] |
+| `userdb_default_fields`, `userdb_override_fields` | Replaced by [[setting,userdb_fields]] |
```

Comment:
should perhaps be passdb { default_fields, override_fields } and same for userdb